streams/src/test/java/org/apache/kafka/streams/kstream/internals/KStreamPeekTest.java

```diff
@@ -66,13 +86,40 @@ public void shouldObserveStreamElements() {
     @Test
     public void shouldNotAllowNullAction() {
         final KStreamBuilder builder = new KStreamBuilder();
-        final KStream<Integer, String> stream = builder.stream(Serdes.Integer(), Serdes.String(), topicName);
+        final KStream<Integer, String> stream = builder.stream(intSerd, stringSerd, topicName);
         try {
             stream.peek(null);
             fail("expected null action to throw NPE");
         } catch (NullPointerException expected) { }
     }
 
+    @Test
+    public void testPrintKeyValueWithName() {
```

Comment:
Now, we will create `KStreamPrint`, so should we have new test class named `KStreamPrintTest` or keep it like current.